auto_nag/scripts/leave_open_no_activity.py

```diff
@@ -40,8 +42,39 @@ def get_mail_to_auto_ni(self, bug):
 
         return None
 
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+        data[bugid] = {"id": bugid, "depends_on": bug["depends_on"]}
+        return bug
+
+    def filter_deps(self, bugs):
+        def handler(bug, data):
+            if bug["resolution"] == "":
+                data.add(bug["id"])
+
+        bugids = set()
+        for info in bugs.values():
+            bugids.update(info["depends_on"])
+        bugids = list(bugids)
```

Comment:
Can you rename the variable `dependencies_ids`, so it's clearer what it refers to?